content/en/docs/reference/command-line-tools-reference/feature-gates/relaxed-environment-variable-validation.md

```diff
@@ -9,5 +9,8 @@ stages:
   - stage: alpha
     defaultValue: false
     fromVersion: "1.30"
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.32"
```

Comment:
Adding `toVersion` to the previous stage would enhance consistency with other feature gate files.

```suggestion
    toVersion: "1.31"
  - stage: beta
    defaultValue: true
    fromVersion: "1.32"
```

Reply:
thanks.